agent/tests/input_files/influx_pipelines.json

```diff
@@ -21,6 +21,22 @@
     "interval": 1200000,
     "uses_schema": false
   },
+  {
+    "source": "test_influx",
+    "pipeline_id": "test_influx_file_transform",
+    "measurement_name": "cpu_test",
+    "values": {"usage_active": "gauge"},
+    "dimensions": {
+      "required": [],
+      "optional": ["cpu", "host"]
+    },
+    "transform": {
+      "file": "/home/influx_transform.csv"
+    },
+    "properties": {"test": "val"},
+    "interval": 1200000,
+    "uses_schema": false
+  },
```

Comment:
can we have a test that uses the query field?

Reply:
Added custom test query quite similar to auto-generated